packages/shared/src/components/post/PostSummaryButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement } from 'react';
+import { useSummaryActivation } from '../../hooks/experiments';
+import { Loader } from '../Loader';
+import { MagicIcon } from '../icons';
+import { ButtonIconPosition, ButtonVariant } from '../buttons/common';
+import { Button, ButtonColor } from '../buttons/Button';
+
+interface PostSummaryButtonProps {
+  summary?: string;
+}
+
+export function PostSummaryButton({
+  summary,
+}: PostSummaryButtonProps): ReactElement {
+  const { shouldShowOverlay, isLoading, onClickSummary } = useSummaryActivation(
+    { summary },
+  );
+
+  if (!shouldShowOverlay) {
+    return null;
+  }
+
+  return (
+    <>
+      <div className="absolute inset-0 bg-gradient-to-t from-background-default from-60% to-overlay-primary-pepper" />
```

Comment:
Just wondering, is this considering the light mode?

Reply:
Yes, since the colors are using the themed classes.

Reply:
Ah nevermind, I thought I saw a set color